assets/together.yaml

```diff
@@ -109,3 +109,71 @@
   prohibited_uses: ''
   monitoring: ''
   feedback: ''
+
+- type: dataset
+  name: RedPajama-Data-1T
+  organization: Together, Stanford, ETH, LAION, Ontocord, EleutherAI
+  description: 'The reproduction of the LLaMA training dataset of over 1.2 trillion tokens'
+  created_date: 2023-04-17
+  url: https://huggingface.co/datasets/togethercomputer/RedPajama-Data-1T
+  datasheet: ''
+  modality: text
+  size: 1.2T tokens
+  sample: []
+  analysis: ''
+  dependencies: []
+  included: ''
+  excluded: ''
+  quality_control: ''
+  access: open
+  license: Apache 2.0
+  intended_uses: ''
+  prohibited_uses: ''
+  monitoring: ''
+  feedback: ''
+
+- type: model
+  name: RedPajama-INCITE-Base-3B-v1
```

Comment:
I think we can merge these into RedPajam-INCITE-Base and note the size as 7B with a description mentioning there is also a 3B model?